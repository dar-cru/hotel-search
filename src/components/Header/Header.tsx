import { FC } from 'react';
import logo from '../../images/qantas-logo.png';

const Header: FC = () => {
  return (
    <header style={{ padding: '20px 0' }}>
      <img src={logo} alt="Qantas logo" width="200px" />
    </header>
  );
};

export default Header;
