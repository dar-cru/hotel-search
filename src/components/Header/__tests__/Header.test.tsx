import { render, RenderResult } from '@testing-library/react';
import Header from '../Header';

describe('Header component', () => {
  it('renders with the header logo', () => {
    const tree: RenderResult = render(<Header />);

    const { container } = tree;
    expect(container).toMatchSnapshot();
  });
});
