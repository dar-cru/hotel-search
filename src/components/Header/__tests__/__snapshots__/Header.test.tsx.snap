// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders with the header logo 1`] = `
<div>
  <header
    style="padding: 20px 0px;"
  >
    <img
      alt="Qantas logo"
      src="qantas-logo.png"
      width="200px"
    />
  </header>
</div>
`;
